packages/dropdowns.next/src/types/index.ts

```diff
@@ -252,3 +226,100 @@ export interface ITagGroupProps {
   /** Provides the current selection */
   selection: IOption[];
 }
+
+export interface IMenuListProps extends HTMLAttributes<HTMLUListElement> {
```

Comment:
I'm not sure the best way to handle this so doc gen ignores it.  Storybook does, but not sure about the website.

Reply:
Always run `garden cmd-docgen` against new components or components with updated doc to verify that the output matches expectations. In this case `yarn garden cmd-docgen --pretty packages/dropdowns.next/src/elements/menu/Menu.tsx` etc.

Reply:
Very handy, thanks. I found a couple errors running this. :+1: